lib/actions.js

```diff
@@ -96,8 +96,8 @@ module.exports = function (config) {
       return reqInstance('DELETE', '/users/' + userUuid + '/transfers/' + transferUuid + '/cancel');
     },
 
-    getBalance: function () {
-      return reqInstance('GET', '/balance');
```

Comment:
Removing functions from an interface is a major version change, does /balance exist? Or are customers just getting a 404 with this?

Reply:
The balance feature was made specifically for Uviba, and was never documented. Hasn't been used since

Reply:
I see though. Should I just add another one then?

Reply:
Ok can we just add a console.error to `getBalance` then instead of removing it entirely? That way we can avoid a major version bump and just make it minor.

Reply:
Good idea. SG